tests/informatics/test_predictor_evaluation_metrics.py

```diff
@@ -6,6 +6,7 @@
 
 @pytest.fixture(params=[
     (RMSE(), "rmse", "RMSE"),
+    (RSquared(), "r_squared", "R^2"),
```

Comment:
This isn't just relevant to this addition, but oughtn't we be importing these parameter lists directly from predictor_evaluation_metrics.py?

Reply:
We could, but then we'd end up asserting `str(metric) == metric.__str__` :p

This is really just boilerplate to satisfy coverage :/